docs/data/pages.ts

```diff
@@ -265,6 +265,10 @@ const pages: MuiPage[] = [
             pathname: '/x/react-date-pickers/adapters-locale',
             title: 'Date localization',
```

Comment:
Just an idea because when stating to read I thaught "What's the difference with the Date localization page?"
```suggestion
            title: 'Format localization',
```

Reply:
For me UTC / Timezone is in some sense a sub-part of "Date localization", but I moved it in a standalone page because otherwise it was super long.
I think "Format localization" would not be as clear, because it impacts stuff outside of the format (the week day labels in the "day" view for example)